Include/object.h

```diff
@@ -122,7 +129,20 @@ whose size is determined when the object is allocated.
 #include <inttypes.h>
 #include <x86intrin.h>
 
+#if defined(__APPLE__)
+#import <mach/mach_time.h>
+#endif
 
+Py_LOCAL_INLINE(uint64_t) fast_get_cycles(unsigned int* t)
+{
+#ifdef __APPLE__
+    return mach_absolute_time();    
+#else
+    return __rdtscp(t);
```

Comment:
So far I don't care about the "t" that comes back from rdtscp.  Can you just throw it away locally in this function?